webvtt.go

```diff
@@ -203,6 +206,50 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	return
 }
 
+func extractVoiceNameAndText(line string) (string, string) {
+	s := scanner.Scanner{}
```

Comment:
I find your implementation a bit complicated which will be a problem to for maintenance.

I'd rather use a regular expression if it works. Something like `(<v([\.\w]*)([\s\w]+)+>)*(.[^<]*)(</v>)*` should work. You get:

- voice name in index 3
- text in index 4

Declare the regexp globally and use the `FindStringSubmatch` method.

If you're not familiar with regexp in go, check out [this regexp](https://github.com/asticode/go-astisub/blob/master/ttml.go#L34)

Reply:
sure. I was inclined to use regexp as well (as in my first commit) but then it seemed a bit complex to figure out the expression to match all possibilities in the w3c spec for voice tag.
thanks for this `(<v([\.\w]*)([\s\w]+)+>)*(.[^<]*)(</v>)*` expression. I will update this PR sooner with the changes.

Reply:
`<v Bob>Incorrect tag?</vi>` the regexp seems to miss this case. It returns Bob. I expected to return empty string since the end tag is not a valid one`</vi>`.

Reply:
Is there a specific reason why you want to catch incorrect finishing tags ?